pkg/pgmodel/end_to_end_tests/main_test.go

```diff
@@ -110,7 +110,12 @@ func performMigrate(t testing.TB, connectURL string) {
 		t.Fatal(err)
 	}
 	defer migratePool.Close()
-	err = Migrate(migratePool, pgmodel.VersionInfo{Version: version.Version, CommitHash: "azxtestcommit"})
+	conn, err := migratePool.Acquire(context.Background())
```

Comment:
Do we need a separate pool? Isn't this always only going to be one connection?

Reply:
Seems like maybe it is a single connection, in which case, the naming is the problem.

Reply:
we don't, but it's easier than refactoring the test more deeply